src/test/regress/sql/view_propagation.sql

```diff
@@ -442,6 +442,13 @@ SELECT create_distributed_table('employees','employee_id');
 SELECT run_command_on_workers($$SELECT count(*) FROM v_test_1$$);
 SELECT run_command_on_workers($$SELECT count(*) FROM v_test_2$$);
 
+-- create and drop temporary view
+CREATE TEMPORARY VIEW temp_view_to_drop AS SELECT 1;
```

Comment:
We seem to have an assertion failure:

```SQL

create materialized view axx.temp_view_to_drop AS SELECT 1;
SELECT 1
Time: 39.294 ms
[local] onderkalaci@postgres:5432-56752=# DROP VIEW axx.temp_view_to_drop;
server closed the connection unexpectedly
    This probably means the server terminated abnormally
    before or while processing the request.
The connection to the server was lost. Attempting reset: 
Failed.
Time: 109.640 ms
 
```

Here:
```C
/*
 * IsViewDistributed checks if a view is distributed
 */
bool
IsViewDistributed(Oid viewOid)
{
	Assert(get_rel_relkind(viewOid) == RELKIND_VIEW);


```

I guess we should be able to add RELKIND_MAT_VIEW here as well?